app/models/role.rb

```diff
@@ -30,12 +30,13 @@ class Role < ApplicationRecord
 
   ROLES = ['admin'].freeze
   PRO_ROLES = ['pro', 'pro_admin'].freeze
+  PROJECT_ROLES = %w[project_owner project_contributor].freeze
 
   def self.allowed_roles
```

Comment:
I don't think we want the `PROJECT_ROLES` here:

![Screenshot 2020-03-25 at 14 27 36](https://user-images.githubusercontent.com/282788/77547251-daa28f80-6ea4-11ea-8eb2-e6c4e541a0fe.png)

Reply:
Oh yes, I was aware of this and have stash somewhere which sorts this - will open a PR

Reply:
Would be good to get this PR up to enable this merge

Reply:
Is up here: https://github.com/mysociety/alaveteli/pull/5603